playwright/_impl/_browser_type.py

```diff
@@ -171,11 +172,24 @@ async def connect(
             self._connection._object_factory,
             transport,
         )
+        await connection._transport.start()
         connection._is_sync = self._connection._is_sync
         connection._loop = self._connection._loop
         connection._loop.create_task(connection.run())
+        obj = asyncio.ensure_future(
```

Comment:
```suggestion
        obj = asyncio.create_task(
```
Use `asyncio.create_task` for consistency for rest of the codebase